bitnami/airflow/templates/statefulset-worker.yaml

```diff
@@ -59,8 +60,29 @@ spec:
           volumeMounts:
             - name: git-cloned-dag-files
               mountPath: /dags
+            - name: dag-git-keys
+              mountPath: /root/.ssh
+        {{- end }}
+        {{- if .Values.additionalRepos.enabled }}
+        {{- range .Values.additionalRepos.repos }}
```

Comment:
This is very inefficient. This is creating a new container for each repo we want to clone. We could clone all the repos using one single container instead.

Reply:
I am not sure.

Reply:
I changed it so it spawns only one pod, and then clones all the repos into that pod, and mounts a volume for each ssh key. 

In saying that, I realize that if all the repos require an ssh key, it will not work correctly. You can only use an ssh key for one repo. So we do need all the containers to be created - one for each repo.